contracts/ref/oss.yml

```diff
@@ -5930,6 +5930,9 @@ paths:
       operationId: GetUsers
       parameters:
       - $ref: '#/components/parameters/TraceSpan'
+      - $ref: '#/components/parameters/Offset'
+      - $ref: '#/components/parameters/Limit'
+      - $ref: '#/components/parameters/After'
```

Comment:
Not sure why this changed. Has nothing to do with notebooksd

Reply:
i believe `make generate` wasn't run on a prior pr.. the `ref` directory is a new addition and testing for it will be added soon